site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+            </button>
+
+            <button type="button" class="btn btn-dark mb-3 order-lg-first">
+              <svg class="me-1" width="1.25rem" height="1.25rem" aria-hidden="true" focusable="false">
+                <use xlink:href="#delete"/>
+              </svg>
+              Reset filter
+            </button>
+          </form>
+
+        </div>
+      </div>
+
+      <div class="col-12 col-lg-9">
+        <div class="d-flex justify-content-between gap-2 mb-4">
+          <div class="dropdown flex-shrink-1" style="width: 146px;">
+            <button class="btn btn-secondary dropdown-toggle justify-content-between w-100" type="button" data-bs-toggle="dropdown" aria-expanded="false" id="results-order">
+              <span class="visually-hidden">Sorted by</span>Newest
+            </button>
+            <ul class="dropdown-menu">
+              <li><button class="dropdown-item active"><span class="visually-hidden">Sort by</span>Newest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Oldest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Cheapest</button></li>
+              <li><button class="dropdown-item"><span class="visually-hidden">Sort by</span>Most expensive</button></li>
+            </ul>
+          </div>
+
+          <div class="d-flex">
+            <button class="btn btn-secondary d-lg-none flex-shrink-1" style="width: 146px;" data-bs-toggle="offcanvas" data-bs-target="#filterOffcanvas" aria-controls="filterOffcanvas">
+              <svg class="me-1" width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#filter"/>
+              </svg>
+              <span class="visually-hidden">Toggle</span>Filter
+            </button>
+
+            <form><div class="btn-group d-none d-md-flex ms-2" role="group">
+              <input type="radio" class="btn-check" name="displayOffers" id="gridDisplay" autocomplete="off" checked>
+              <label class="btn btn-icon btn-no-outline" for="gridDisplay" title="Grid">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#grid"/>
+                </svg>
+                <span class="visually-hidden">Grid display</span>
+              </label>
+              <input type="radio" class="btn-check" name="displayOffers" id="listDisplay" autocomplete="off">
+              <label class="btn btn-icon btn-no-outline" for="listDisplay" title="List">
+                <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                  <use xlink:href="#list"/>
+                </svg>
+                <span class="visually-hidden">List display</span>
+              </label>
+            </div></form>
+          </div>
+        </div>
+
+        <p class="pt-md-3 h4 mb-3">
```

Comment:
I would add `aria-live="polite" aria-atomic="true"` on this section to ensure screen readers user that it will be updated according to their search.